wiki/Restricted_mode/en.md

```diff
@@ -0,0 +1,9 @@
+---
+stub: true
+---
+
+# Restricted mode
+
+**Restricted mode** is a state that is automatically applied to accounts that are suspected of breaking the [Terms of Service](https://osu.ppy.sh/legal/terms). Submitted scores of gameplay will be hidden from the [leaderboards](/wiki/Ranking) as a result of this.
+
+For more info, see [What is "restricted" mode, exactly?](/wiki/Help_Centre#restriction).
```

Comment:
Really weird that the help centre would give "more info" than the restricted mode article itself.

Reply:
This was a glossary term, but since it may cause some confusion/messy linking, I'll just remove this article instead.

Reply:
I would've opposed removing it until I saw https://osu.ppy.sh/help/wiki/Help_Centre/Account_Restrictions. Doesn't seem needed anymore.